src/Microsoft.DotNet.Cli.Utils/CommandResolution/IPlatformCommandSpecFactory.cs

```diff
@@ -0,0 +1,23 @@
+using System;
```

Comment:
This is only not used in ProjectTools/ProjectDependencies because they don't support .cmd extensions.